src/Styleguide/Artwork/Gallery.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react"
+import { Flex } from "../Elements/Flex"
+import { Avatar, Button } from "../Elements"
+import { Serif, Sans } from "@artsy/palette"
+
+interface GalleryProps {
+  src: string
+  headline: string
+  subHeadline: string
+}
+
+export class Gallery extends React.Component<GalleryProps> {
+  render() {
+    return (
+      <Flex>
+        <Avatar src={this.props.src} size="100px" />
+        <Flex flexDirection="column" justifyContent="center" ml={4}>
+          <Serif size="5t">{this.props.headline}</Serif>
+          <Sans size="2">{this.props.subHeadline}</Sans>
+          <Button variant="outline" size="small" mt={3} width="80px">
+            Follow
+          </Button>
```

Comment:
😄

Reply:
I wanted to pull the `width` size from space, but it wasn't really working out for me. Didn't spent _too_ much time trying to figure it out though. I'll likely revisit it shortly.